### DISCUSSION TITLE
Disclosure in a callout?

### DISCUSSION THREAD

**Discussion by mma1504** [twilio-labs/paste#3354] [General] [answered]
Are there any restrictions on using a disclosure within a callout? 

I had a situation at content office hours today where we're showing an error message in a callout. We were writing the message from the Segment perspective (basically, we couldn't sync with the warehouse), but we also have access to the error message sent from the warehouse. One solution we discussed was to show the simple Segment error message (the thing couldn't happen. we'll try again in an hour), but also provide the warehouse error message behind a disclosure in case someone wanted to investigate the issue further.

**Comment by loreina** [comment, marked as answer]
Hi @mma1504,

There aren't any technical restrictions on using a Disclosure within a Callout, though it might not have the most appealing visual treatment.

We've come across a similar use case with an Alert and having a large error message: https://github.com/twilio-labs/paste/discussions/2438. Our recommendation there was to include a button that opens a modal with the rest of the error messages if there's more than 3. Maybe this'll be helpful for your use case. Let us know what you end up going with and if it's guidance we can add to our docs site. Thanks!
